ansible/README.md

```diff
@@ -2,38 +2,45 @@
 
 These two ansible playbooks automate the deployment and cleanup of Sheepdog and WebGoat7 to quickly setup a demo environment using the Contrast Security Secure DevOps Platform.
 
-**Note:** You must download and copy your contrast.jar file to the same directory where webgoat7 and sheepdog are installed. The default directory is ``` ~/webgoat7```.
-
 ## To use this playbook, follow the instructions below
 
-### 1. Clone this github repo.
+### Create a '~.contrast.jar file'
+The playbook relies on API credentials in this file.
```

Comment:
Selfishly, it would be great to be able to configure multiple sets of credentials and then pass in `--profile` (or similar) to note which set to use.